packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -359,6 +360,12 @@ export interface FunctionOptions extends EventInvokeConfigOptions {
    * @default Architecture.X86_64
    */
   readonly architecture?: Architecture;
+
+  /**
+   * Sets the runtime management configuration for a function's version.
+   * @default Auto
+   */
+  readonly runtimeManagement?: RuntimeManagement;
```

Comment:
```suggestion
  readonly runtimeManagementMode?: RuntimeManagementMode;
  
  readonly runtimeManagementVersionArn?: string;
```

Reply:
Implemented as specified in `.manual(arn)` of RuntimeManagementMode.